.github/workflows/compile_examples.yml

```diff
@@ -1,6 +1,10 @@
 name: Compile Examples
 
-on: [pull_request]
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
```

Comment:
Is there a reason to have the CI running on every push?

I think it is not necessary to use the CI for this, it is more sustainable to execute `make` locally...

Reply:
But it could be interesting to build it on the main branch every week, so that updates to the container avr-gcc get tested against this automatically. Alternatively replace `:latest` with a tag, but this repo is not going to be maintained as often as modm.

Reply:
For the moment (but I'm surely flexible to change this later) I have retained run CI on branch pushes.